### PR TITLE
Simple script to check Vagrant devstack services are online

### PR DESCRIPTION
**Pull request by macdiesel**

### CODE REVIEW

File: tubular/scripts/vagrant_devstack_healthcheck.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+    """
+    failed_services = []
+    for service in SERVICES:
+        if not service.enabled:
+            pass
+        url = 'http://{netloc}:{port}/{path}'.format(netloc=service.host, port=service.port, path=service.path)
+        try:
+            if not is_service_operational(url):
+                failed_services.append((service, 'Service running but returns non 200 health response code'))
```
Comment (doctoryes) [inline tubular/scripts/vagrant_devstack_healthcheck.py]: Nit: Would it be useful to return the actual response code here?

Comment (doctoryes) [inline tubular/scripts/vagrant_devstack_healthcheck.py]: Also, `non-200`.

File: tubular/scripts/vagrant_devstack_healthcheck.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        except URLError:
+            failed_services.append((service, 'Connction Refused'))
```
Comment (doctoryes) [inline tubular/scripts/vagrant_devstack_healthcheck.py]: `Connection refused.`

File: tubular/scripts/vagrant_devstack_healthcheck.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    Check to see if a service is available for a given URL.
+
+    Args:
+        url(str): the url for the service to be checked
+
+    Returns:
+        bool: True when the response code is 200
+
+    """
+    return bool(urlopen(url).code == 200)
```
Comment (doctoryes) [inline tubular/scripts/vagrant_devstack_healthcheck.py]: Is the outer `bool` method necessary here?

**Comment by macdiesel** [comment]
Ready for re-review.

**Review by doctoryes — APPROVED** [review]

**Pull request merged**